deploy/tests/frontmatter.rb

```diff
@@ -0,0 +1,74 @@
+require 'yaml'
+def loadconfig()
+	config = YAML.load_file('_config.yml')
+end
+@config = loadconfig
```

Comment:
This is good to know, @konklone thanks for the tip. I was looking for a cleaner way of expressing this.